File: mobile-client/android/Contract/Product/Detail/ProductDetailPayload.kt
package com.smartresponsor.mobile.client.contract.product.detail

/**
 * Marketing America Corp. Oleksandr Tishchenko
 */
data class ProductDetailPayload(
    val productId: String,
    val title: String,
    val description: String?,
    val priceLabel: String?,
    val mediaUrls: List<String>,
    val vendorLabel: String?,
)

File: client/ios/Contract/Product/Detail/ProductDetailPayload.swift
import Foundation

// Marketing America Corp. Oleksandr Tishchenko
public struct ProductDetailPayload: Sendable, Equatable {
    public let productId: String
    public let title: String
    public let description: String?
    public let priceLabel: String?
    public let mediaUrls: [String]
    public let vendorLabel: String?
}
